### PR TITLE
Add some information about ariakit to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sliminas**
## Description of the change

Add some information about ariakit to CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Contributing
 
+## ariakit
+
+We're using [@ariakit-react](https://ariakit.org) to build many of the components in Pluto.
+Currently we're still using an outdated version of it (formerly named
+[reakit](https://reakit.io/docs/get-started/)). For some of the components you might to look at the
+old versions documentation until we upgraded to the new version (
```
Comment (blumaa) [inline CONTRIBUTING.md:8]:
```suggestion
old version's documentation until we upgraded to the new version)
```

**Review by blumaa — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Contributing
 
+## ariakit
+
+We're using [@ariakit-react](https://ariakit.org) to build many of the components in Pluto.
+Currently we're still using an outdated version of it (formerly named
+[reakit](https://reakit.io/docs/get-started/)). For some of the components you might to look at the
```
Comment (padulafacundo) [inline CONTRIBUTING.md:7]:
```suggestion
[reakit](https://reakit.io/docs/get-started/)). For some of the components you might want to look at the
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (padulafacundo) [reply CONTRIBUTING.md:8]: Also
```suggestion
old versions documentation until we upgrade to the new version (
```
right?

**Review by padulafacundo — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (sliminas) [reply CONTRIBUTING.md]: This specifically refers to the point in time after we upgraded. So the upgrade event should be in the past.

**Pull request merged**